chrome/content/brief-overlay.js

```diff
@@ -135,14 +135,11 @@ var Brief = {
             }.bind(this))
         }
 
-        if (this.toolbarbutton)
```

Comment:
The initUnreadCounter function is not accessing this.toolbarbutton. In initUnreadCounter the context menus is accessed without any additional checks.
If the checks are kept, initUnreadCounter must also be called when the toolbarbutton is loaded.